### PR TITLE
Make the node outputs some logs when the node process stopped by Unix signals

### PR DESCRIPTION
**Pull request by rantan**
```
[2019-12-25T08:19:26Z INFO  tapyrus_signer::signer_node] node start. NodeState: Member { block_key: None, shared_block_secrets: {}, block_shared_keys: None, candidate_block: None }, node_index: 2, master_index: 0
[2019-12-25T08:19:26Z TRACE tapyrus_signer::signer_node] No stop signale
[2019-12-25T08:19:26Z TRACE tapyrus_signer::signer_node] No stop signale
[2019-12-25T08:19:27Z TRACE tapyrus_signer::signer_node] No stop signale
[2019-12-25T08:19:27Z TRACE tapyrus_signer::signer_node] No stop signale
[2019-12-25T08:19:27Z TRACE tapyrus_signer::signer_node] No stop signale
[2019-12-25T08:19:28Z TRACE tapyrus_signer::signer_node] No stop signale
[2019-12-25T08:19:28Z TRACE tapyrus_signer::signer_node] No stop signale
^C[2019-12-25T08:19:28Z INFO  tapyrus_signer::signer_node] Signer Node was stopped by SIGINT
```

### CODE REVIEW

File: src/signer_node.rs [hunk 1]
```diff
@@ -197,6 +203,18 @@ impl<T: TapyrusApi, C: ConnectionManager> SignerNode<T, C> {
                 },
                 None => {}
             }
+
+            // Unix Signal handler
+            match unix_stop_signal_handler.load(Ordering::Relaxed) {
+                0 => {
+                    log::trace!("No stop signale");
```
Comment (azuchi) [inline src/signer_node.rs]: I think this `No stop` log is not necessary, it will be garbage log.

File: src/signer_node.rs [hunk 2]
```diff
@@ -178,6 +179,11 @@ impl<T: TapyrusApi, C: ConnectionManager> SignerNode<T, C> {
         self.round_timer.start().unwrap();
         // get error_handler that is for catch error within connection_manager.
         let connection_manager_error_handler = self.connection_manager.error_handler();
+
+        // Add signal handler
+        let unix_stop_signal_handler =
+            set_stop_signal_handler().expect("Failed to register signal handler.");
+
```
Comment (azuchi) [inline src/signer_node.rs]: In this scope, no log is output before connecting redis. Is it better to control on the main function in `node.rs`?

File: src/signer_node.rs (hunk 1 above)
Comment (rantan) [reply src/signer_node.rs]: I think it is ok because of  TRACE level log.

File: src/signer_node.rs (hunk 2 above)
Comment (rantan) [reply src/signer_node.rs]: it looks good. i will change.

**Pull request merged**